Maven cannot fetch transitive dependencies of `browserup-proxy-core:2.2.5`

**Describe the bug**
Maven cannot fetch transitive dependencies of `browserup-proxy-core`.

**To Reproduce**
Steps to reproduce the behaviour:
1. Create a Maven project with `browserup-proxy-core` dependency:

```xml
    <dependency>
      <groupId>com.github.valfirst.browserup-proxy</groupId>
      <artifactId>browserup-proxy-core</artifactId>
      <version>2.2.5</version>
    </dependency>
```
2. Run `mvn dependency:tree`
3. Look in logs:
4. Apparently, the problem was introduces in commit 5125e38bf3e526e2e07232. 
```
[WARNING] The POM for com.github.valfirst.browserup-proxy:browserup-proxy-core:jar:2.2.5 is invalid, transitive dependencies (if any) will not be available: 3 problems were encountered while building the effective model for com.github.valfirst.browserup-proxy:browserup-proxy-core:2.2.5
[ERROR] 'dependencies.dependency.version' for com.fasterxml.jackson.core:jackson-core:jar is missing. @ 
[ERROR] 'dependencies.dependency.version' for com.fasterxml.jackson.core:jackson-databind:jar is missing. @ 
[ERROR] 'dependencies.dependency.version' for com.fasterxml.jackson.core:jackson-annotations:jar is missing. @ 
```
5. It seems the problem happens only with Maven. Gradle works like a charm. 


**Please complete the following information:**
 - Version `browserup-proxy-core:2.2.5` and `2.2.4`.
 - Version `2.2.3` doesn't have this problem.


**Additional context**
See error description in https://github.com/asolntsev/browserup-dependency-bug

--- a/browserup-proxy-core/build.gradle
+++ b/browserup-proxy-core/build.gradle
@@ -64,10 +64,9 @@ dependencies {
     implementation 'javax.xml.bind:jaxb-api:2.3.1'
 
     implementation 'org.awaitility:awaitility:4.2.0'
-    implementation platform('com.fasterxml.jackson:jackson-bom:2.14.0')
-    implementation 'com.fasterxml.jackson.core:jackson-core'
-    implementation 'com.fasterxml.jackson.core:jackson-databind'
-    implementation 'com.fasterxml.jackson.core:jackson-annotations'
+    implementation "com.fasterxml.jackson.core:jackson-core:${jacksonVersion}"
+    implementation "com.fasterxml.jackson.core:jackson-databind:${jacksonVersion}"
+    implementation "com.fasterxml.jackson.core:jackson-annotations:${jacksonVersion}"
 
     implementation "com.google.guava:guava:${guavaVersion}"
     implementation 'com.jcraft:jzlib:1.1.3'

--- a/build.gradle
+++ b/build.gradle
@@ -31,6 +31,7 @@ subprojects {
     ext {
         bcpVersion = '1.72'
         guiceVersion = '4.2.3'
+        jacksonVersion = '2.14.1'
         jerseyVersion = '2.32'
         jettyVersion = '9.4.49.v20220914'
         log4jVersion = '2.19.0'

